Powershell/InstallHuntress.powershellv2.ps1

```diff
@@ -784,10 +803,10 @@ function logInfo {
 # In the past we copied to the users temp folder, difficult to find on machines with lots of profiles. Solved this by always placing the log in the normal Huntress folder.
 function copyLogAndExit {
     Start-Sleep 1
-    $logLocation = $DebugLog
-    $agentPath   = getAgentPath
-    Copy-Item -Path $logLocation -Destination $agentPath -Force
-    Write-Output "$($Debuglog) copied to $(getAgentPath)"
+    $agentPath = getAgentPath
+    $logLocation = Join-Path $agentPath "HuntressPoShInstaller.log"
```

Comment:
Just throwing this out there... when is $logLocation being used? It looks like it's just assigned to the Join-Path and not used after that

Reply:
Updated it